parsl/dataflow/usage_tracking/usage.py

```diff
@@ -185,7 +185,7 @@ def construct_end_message(self):
 
         site_count = len([x for x in self.dfk.config.executors if x.managed])
 
-        failed_states = (States.failed, States.dep_fail)
+        failed_states = FINAL_FAILED_STATES
```

Comment:
I slightly prefer search-and-replace `failed_states -> FINAL_FAILED_STATES` but don't want to hold up the PR for that